c10/core/Storage.h

```diff
@@ -46,22 +42,16 @@ struct C10_API Storage {
   // Legacy constructor for partially initialized (dtype or memory) storages
   // that can be temporarily created with Caffe2 APIs. See the note on top of
   // TensorImpl.h for details.
-  static Storage create_legacy(at::Device device, caffe2::TypeMeta data_type) {
+  static Storage create_legacy(at::Device device) {
```

Comment:
Need to verify that call site sets data type on tensor eventually. **THIS IS PROBABLY THE CAUSE OF THE CI FAILURES**

Reply:
Actually there is a type signature change here so we should see the call sites already.